test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/Source/DynamicAttributeTagHelpers.cshtml

```diff
@@ -0,0 +1,14 @@
+﻿@addTagHelper "something, nice"
+
+<input unbound="prefix @DateTime.Now" />
```

Comment:
We cover all of those cases in the Basic, Complex and TagHelpersInSection test files.